stx-rosetta.Dockerfile

```diff
@@ -1,147 +1,124 @@
-### Build blockstack-core-sidecar API
-FROM node:lts-buster as build
-
-ARG API_TAG=v0.29.4
-
-RUN apt-get -y update && apt-get -y install openjdk-11-jre-headless
 
+#######################################################################
+## Build the stacks-blockchain-api
+FROM node:lts-buster as stacks-blockchain-api-build
+ARG STACKS_API_REPO=blockstack/stacks-blockchain-api
+ENV STACKS_API_REPO=${STACKS_API_REPO}
```

Comment:
I think this ENV var can be removed. `${STACKS_API_REPO}` will just use the `ARG` like it's an ENV var. The ENV var directive should be used when the env variable needs to be available in the resulting image.

Reply:
unless the repo name changes, in which case this is a single change or a build-arg vs multiple changes in the file

Reply:
I'm saying you can delete just this ENV var and the image will work the same as it did before. There's no benefit to declaring an ARG then setting an ENV var to it if the env var won't be used in the final image because it's redundant. If the repo name changes, you'd update the ARG default value.